rectools/models/sasrec.py

```diff
@@ -45,7 +45,7 @@ def get_all_embeddings(self) -> torch.Tensor:
 
     @property
     def device(self) -> torch.device:
```

Comment:
I think we should drop all `device` properties. And stop doing `to(device)` and do `to(<tensor>)` instead. Otherwise we can catch problems with lighting multi-device training. 
Let's change it to lightning advices for device agnostic code https://pytorch-lightning.readthedocs.io/en/1.0.8/multi_gpu.html

We can do it in this PR or in the next one

Reply:
Fixed